cmd/vic-machine/configure/configure.go

```diff
@@ -321,6 +321,8 @@ func (c *Configure) Run(clic *cli.Context) (err error) {
 	defer cancel()
 
 	validator, err := validate.NewValidator(ctx, c.Data)
+	defer validator.Session.Logout(ctx)
```

Comment:
Can you add a test that uses an invalid target and/or credentials?  afiact, every `defer Logout` in this PR would panic in that case.. looks like the `defer Logout`'s should be after the err != nil checks.